dev-docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,142 @@
+
+Now, with the boot parameter in place, test that a xenial container can start and reach the network.
+```bash
+lxc launch ubuntu-daily:xenial testx
+lxc shell testx
+# now you should be inside the container
+ping -c 3 ubuntu.com
+# the ping command should succeed
+exit
+# now you should be back on your host
+lxc delete --force testx
+```
+
+> **Note**
+> Docker can interfere with lxd container networking. If you need docker installed alongside lxd, follow the guidance in the lxd documentation to ensure that docker doesn't break lxd networking: https://documentation.ubuntu.com/lxd/en/latest/howto/network_bridge_firewalld/#prevent-connectivity-issues-with-lxd-and-docker
+
+## Building Ubuntu Pro Client for testing
+
+In order to install a local version of Ubuntu Pro Client in a lxd container, we need to build a `deb` package. We have a script that will set up the environment necessary to build debs for any target Ubuntu release.
+
+At time of writing, Ubuntu Pro Client supports the Ubuntu releases in this example command. You may need to adjust the command in the future as Ubuntu releases come and go.
+
+This command also assumes you are on an `amd64` system. You will have to adjust the command accordingly if you are not.
+
+```bash
+env RELEASES="xenial bionic focal jammy mantic noble" ARCHS="amd64" bash tools/setup_sbuild.sh
+```
+
+This command will take some time. It sets up schroots for each release with the dependencies of Ubuntu Pro Client pre-installed. This will make building the deb packages for each release go faster. As time goes by, Ubuntu releases get updates which need to be installed for each build; you can re-run the setup_sbuild.sh script and it will update the schroots to keep your pro-client builds fast.
+
+You will also need to run the following to ensure your user can use the schroots.
+```bash
+sudo sbuild-adduser $USER
+newgrp sbuild
+```
+
+After that is complete, try out a xenial build.
+```bash
+./tools/build.sh xenial
+```
+
+## Configuring pycloudlib
+
+We use [`pycloudlib`](https://github.com/canonical/pycloudlib) to manage instances on clouds for our behave tests, and local lxd containers are treated as a "cloud" by pycloudlib.
+
+To get started, we just need a basic configuration of pycloudlib. Copy the contents of the pycloudlib.toml.template from the source repository and save it to `~/.config/pycloudlib.toml` on your machine.
+```bash
+wget https://raw.githubusercontent.com/canonical/pycloudlib/main/pycloudlib.toml.template -O ~/.config/pycloudlib.toml
+```
+
+## Run a behave test
+
+```bash
+tox -e behave -- features/config.feature -D releases=xenial -D machine_types=lxd-container
+```
+
+All of the arguments after the `--` are passed to behave. In this case, we're telling behave to only run the `config.feature` test, and to filter the tests in that file to only those for xenial lxd containers. Note that the `-D` options are specific to our behave tests and are not behave options.
```

Comment:
I'd say it would be nice to highlight, maybe here, all the `-D` options we support, plus the behave native `-i` and `-n`, for usefulness' sake

Reply:
*looks up `-i`* - that's cool

I agree we need documentation on these, but I'm not sure if it makes sense to put it all in this tutorial. I'm pondering creating a "how to work with behave" and/or "pro behave reference manual" type document that is a bit more comprehensive instead.